openslides_backend/action/actions/structure_level_list_of_speakers/update.py

```diff
@@ -36,10 +36,19 @@ def update_instance(self, instance: dict[str, Any]) -> dict[str, Any]:
                         f"Cannot set initial_time and {field} at the same time."
                     )
             instance["remaining_time"] = instance["initial_time"]
```

Comment:
`current_start_time` also needs to be changed if `remaining_time` is changed by changing the `initial_time`